public/js/app.js

```diff
@@ -1 +1,60 @@
 $(document).foundation()
+
+$( document ).ready(function() {
+  
+  $('#mailchimp-signup').hide();
+  
+});
+
+whatIsTrigger = false;
+
+function clickWhatIsHackathon() {
+  if (whatIsTrigger) {
+    //turn off
+    $('#about').css("background-color","#EADFD8");
```

Comment:
The fade in fade out logic could be separated into a function of its own. 

https://en.wikipedia.org/wiki/Don%27t_repeat_yourself